CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 0.12.0-rc.1
```

Comment:
TODO: add new unreleased version\

Q: should we do it here or should the released version have a "clean changelog"